USAGE.md

```diff
@@ -8,100 +8,281 @@ Baritone commands can also by default be typed in the chatbox. However if you ma
-- `allowBreak`
-- `allowSprint`
-- `allowPlace`
-- `allowParkour`
-- `allowParkourPlace`
-- `blockPlacementPenalty`
-- `renderCachedChunks` (and `cachedChunksOpacity`) <-- very fun but you need a beefy computer
-- `avoidance` (avoidance of mobs / mob spawners)
-- `legitMine`
-- `followRadius`
-- `backfill` (fill in tunnels behind you)
-- `buildInLayers`
-- `buildRepeatDistance` and `buildRepeatDirection`
-- `worldExploringChunkOffset`
-- `acceptableThrowawayItems`
-- `blocksToAvoidBreaking`
-- `mineScanDroppedItems`
-- `allowDiagonalAscend`
-
-
-
-
-# Troubleshooting / common issues
+
+
+
+# Process Management
+
+Baritone uses a process system where only one main process can be active at a time. Available processes include:
+
+- **Builder Process** - Building schematics and structures
+- **Mine Process** - Mining specific blocks or quantities  
+- **Farm Process** - Automatic farming operations
+- **Follow Process** - Following players, entities, or picking up items
+- **Explore Process** - World exploration and chunk discovery
+- **Elytra Process** - Automated elytra flying in the Nether
+- **Custom Goal Process** - Manual goal setting and pathfinding
+- **Get To Block Process** - Pathfinding to specific block types
+
+Use `proc` to see which process is currently active and get information about its state.
+
+# Troubleshooting / Common Issues
 
 ## Why doesn't Baritone respond to any of my chat commands?
-This could be one of many things.
+This could be several things:
+
+1. **Check Installation**: Make sure Baritone is installed. It should create a `baritone` folder in your Minecraft directory.
+
+2. **Check Prefix Settings**: Verify you're using the correct prefix:
+   - Default prefix is `#` (e.g., `#help`)
+   - Some clients like Impact use `.b` as an alias
+   - Check if direct chat control is enabled with `#chatControl` and `#prefixControl`
+
+3. **Client-Specific Issues**: Some clients like Impact disable direct chat control by default. This is a **saved setting** that persists across different clients. If `chatControl` was turned off in Impact, it will remain off in other clients too.
+   - Solution: Use the `#` prefix or edit `baritone/settings.txt` in your Minecraft folder
+   - Remove the line `chatControl false` and restart your client
+
+## Common Command Issues
+
+**Q: Why can I do `.goto x z` in Impact but not elsewhere?**  
+A: That's a custom command added by Impact, not a Baritone command. Use `#goto x z` instead.
+
+**Q: Why can I do `-path to x z` in KAMI but not elsewhere?**  
+A: That's a KAMI-specific command. The Baritone equivalent is `#goto x z`.
+
+## Performance Issues
+
+**Q: Baritone is running slowly or causing lag**  
+A: Try these settings:
+- Disable `renderCachedChunks` if enabled
+- Increase `primaryTimeoutMS` to give more time for pathfinding
+- Disable `avoidance` (it impacts performance significantly)
+- Reduce `followRadius` if using follow commands
```

Comment:
Increasing the primary timeout either has no effect or makes Baritone spend more CPU time before finishing a path.

Decreasing `followRadius` shouldn't be a problem unless it was set really high. If that's wrong, let me know and I'll fix it.

What can indeed cause severe lag is the world scanner when either for lots of blocks on short intervals. If `mineScanUpdateInterval` is lower than the duration of a single scan this will outright freeze the game.